src/modules/vendor/components/programs/cards/templates/OrderTheSequence.vue

```diff
@@ -1,61 +1,42 @@
 <template>
   <div class="container">
-    <ni-input class="q-mb-lg" caption="Question" v-model="card.question" required-field :disable="disableEdition"
+    <ni-input class="q-mb-lg" caption="Question" v-model="card.question" required-field
       @focus="saveTmp('question')" @blur="updateCard('question')" :error="v$.card.question.$error" type="textarea"
       :error-message="errorMsg('question')" />
-    <div v-for="(orderedAnswers, i) in card.orderedAnswers" :key="i" class="answers">
-      <ni-input :caption="`Réponse ${i + 1}`" v-model="orderedAnswers.text"
-        @focus="saveTmp(`orderedAnswers[${i}].text`)" @blur="updateTextAnswer(i)" :disable="disableEdition"
-        :error="getError('orderedAnswers', i)" class="input" :required-field="answerIsRequired(i)" />
-      <ni-button icon="delete" @click="validateAnswerDeletion(i)" :disable="disableAnswerDeletion" />
+    <div class="q-mb-lg">
+      <div v-for="(orderedAnswers, i) in card.orderedAnswers" :key="i" class="answers">
```

Comment:
a quoi sert cette div ? on pourrait mettre le `v-for` sur le `ni-input` d'en dessous et supprimer cette balise et la classe associée ?

Reply:
done